panther_analysis_tool/main.py

```diff
@@ -56,8 +59,14 @@
 )
 
 from panther_analysis_tool.data_model import DataModel
+from panther_analysis_tool.destination import FakeDestination
 from panther_analysis_tool.enriched_event import PantherEvent
+from panther_analysis_tool.exceptions import (
+    FunctionReturnTypeError,
```

Comment:
For return type errors we now return a specific error type, so that we can detect and display that to the user.